docs/csharp/debugging.md

```diff
@@ -200,17 +200,17 @@ With the C# Dev Kit extension installed, the debugger allows you to apply C# cod
 
 ![Hot Reload displayed in the debugging toolbar](images/debugging/hotreload-toolbar.png)
 
-In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [User Settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
+In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [user settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
```

Comment:
The header is "User settings" so you can use that casing in the link

Reply:
got it, I'll correct the other references to that then